ci/slab.toml

```diff
@@ -40,6 +40,16 @@ environment_name = "canada"
 image_name = "Ubuntu Server 22.04 LTS R535 CUDA 12.2"
 flavor_name = "n3-H100x1"
 
+[backend.hyperstack.2-h100]
+environment_name = "canada"
+image_name = "Ubuntu Server 22.04 LTS R535 CUDA 12.2"
+flavor_name = "n3-H100x2"
+
+[backend.hyperstack.4-h100]
```

Comment:
I suppose this profile is put there to quickly hack the CI to run onto 4xH100 ?

Reply:
Yep